### PR TITLE
Update DigitalOcean addons

### PR DESCRIPTION
**Pull request by rifelpet**
The DO e2e presubmits will likely fail with the same ongoing failure as the periodics.

### CODE REVIEW

File: upup/models/cloudup/resources/addons/digitalocean-csi-driver.addons.k8s.io/k8s-1.22.yaml.template [hunk 1]
```diff
@@ -1088,15 +1088,11 @@ spec:
...
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.1
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.8.0
           args:
             - "--v=5"
             - "--csi-address=$(ADDRESS)"
             - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
-          lifecycle:
-            preStop:
-              exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/dobs.csi.digitalocean.com /registration/dobs.csi.digitalocean.com-reg.sock"]
```
Comment (rifelpet) [inline upup/models/cloudup/resources/addons/digitalocean-csi-driver.addons.k8s.io/k8s-1.22.yaml.template:1099]: https://github.com/digitalocean/csi-digitalocean/pull/487

**Comment by rifelpet** [comment]
/skip

The only failure is the same test failing in the periodic.
/cc @hakman

**Review by hakman — APPROVED** [review]

**Comment by hakman** [comment]
> /skip
> 
> The only failure is the same test failing in the periodic. /cc @hakman

@rifelpet I don't think /skip works

**Pull request merged**